app/models/timeline_entry.rb

```diff
@@ -2,4 +2,14 @@ class TimelineEntry < ApplicationRecord
   belongs_to :coronavirus_page
   validates :heading, presence: true, length: { maximum: 255 }
   validates :content, presence: true
+  after_create :set_position
+
+  def set_position
+    coronavirus_page.timeline_entries.each do |timeline_entry|
+      new_position = timeline_entry.position.present? ? timeline_entry.position + 1 : 1
+      timeline_entry.update!(position: new_position)
+    end
```

Comment:
Thanks! This is much nicer. I don't think adding a new timeline entry should update the "updated_at" time of the existing entries if it's just the position that's changed, so this is better.

Will update and fix up.

Reply:
Great stuff - a thing to bear in mind is that this will cause an imbalance with https://github.com/alphagov/collections-publisher/pull/1253/files#diff-abcc24eb29072787841a2fdc3b350199c9a9123251ca8d8b406631c22bbe4979R16 which will update the timetamp. You can change that to `update_column(:position, index)` to avoid that.